homeassistant/components/steam_online/sensor.py

```diff
@@ -70,7 +71,9 @@ def __init__(self, account, steamod):
         self._steamod = steamod
         self._account = account
         self._profile = None
-        self._game = self._state = self._name = self._avatar = None
+        self._game = (
+            self._state
+        ) = self._name = self._avatar = self._last_online = self._level = None
```

Comment:
That looks pretty wired. I guess that this is how `black` handles it.

I would prefer if it's more human-readable. Perhaps split it.